docs/source.rst

```diff
@@ -18,77 +18,54 @@ Source Guide
    methods, including email to Freedom of the Press Foundation.
 
 What is SecureDrop?
----------------------------
+-------------------
 
-Dozens of news organizations — from *ProPublica* to *The New York Times* — use
-SecureDrop to accept tips securely and anonymously. You can reach out and share
-files, and messages, but for real anonymity, it’s important to take some extra
-precautions. This resource will describe things you can do to help protect your
-anonymity when using SecureDrop.
+SecureDrop is a tool that news organizations and NGOs use that enables secure and anonymous communication between whistleblowers and journalists. No personal information is collected; SecureDrop's servers are encrypted, and SecureDrop is not a “cloud” service. If you don’t have sensitive information to send to a news organization, it may be okay to use a traditional computer when reaching out. 
 
-Before moving ahead, note that your Internet Service Provider, or ISP (e.g.,
-Comcast), may already have a record of your visit to this website,
-docs.securedrop.org. Likewise, any related activity should be conducted outside
-of your workplace; if you are reading this page on a workplace device or
-network, they may also have a record of that.
+SecureDrop can accept both messages and individual file uploads (up to 500MB). If you have multiple files to submit, you may do that As a source, you can also return to receive follow-up correspondence with an organization, or to send additional information. Dozens of news organizations — from *ProPublica* to *The New York Times* — use SecureDrop to accept tips securely and anonymously.
 
-Here are some things you can do to further minimize risk.
+To truly protect your anonymity, it is important for you to take some extra precautions in advance. This resource will describe things you can do to help protect your anonymity when using SecureDrop. Note that your Internet Service Provider, or ISP (e.g., Comcast, Cox, Xfinity, Wave, etc), may already have a record of your visit to this website, docs.securedrop.org. 
```

Comment:
Comcast and Xfinity are the same thing (Xfinity is just a rebranding they did because everyone hated Comcast); for name recognition purposes we could say "Comcast/Xfinity".

Reply:
:bulb: I had no idea. :+1: